### PR TITLE
Update rag manifest that should fix the "manifest unknown" issue while image build

### PR DESCRIPTION
**Pull request by mabulgu**
## Description

PR for updating rag manifest in the Containerfil. this should fix the "manifest unknown" issue with the push pipeline.

## Type of change

- [ ] Refactor
- [ ] New feature
- [x] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [ ] Bump-up library or tool used for development (does not change the final image)
- [x] CI configuration change
- [x] Konflux configuration change

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
This can be tested after merging to main.

### CODE REVIEW

File: Containerfile [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # vim: set filetype=dockerfile
-ARG LIGHTSPEED_RAG_CONTENT_IMAGE=quay.io/openshift-lightspeed/lightspeed-rag-content@sha256:8adbb6387204d39fb58eda8d7cdaab6407bb8e09d3050854588f89cd0af62077
-
+ARG LIGHTSPEED_RAG_CONTENT_IMAGE=quay.io/openshift-lightspeed/lightspeed-rag-content@sha256:cbc9b0ea7c96fe400f2f8a8470976dd2c953c8e6239dc61b6aba2fad267363cd
```
Comment (jameswnl) [inline Containerfile:2]: This is a `ARG`.  Please update it in the workflows, e.g. the push workflow is: https://github.com/ansible/ansible-chatbot-service/blob/main/.tekton/ansible-chatbot-service-push.yaml#L269

This ContainerFile will be a generic one on the upstream, we use the `ARG` to control what we need to use

Comment (mabulgu) [reply Containerfile:2]: yeah but only if we wanted to override the value with a custom one. in this case I am suspecting of a meta issue and by default this should be running without a problem. 

What if someone wants to build the image without the pipeline? for the dev environmetn this should be sth already valid as well so I will insist on keeping the change. If this doesnt work, then I can revert it back to tthe upstream one as I will look at the problem from a different angle.

Comment (jameswnl) [reply Containerfile:2]: The default in the container file should be empty. And a local build can supply the value. Build ARG is part of the spec supported by both podman and docker.

That said I'll lgtm this for you

**Review by jameswnl — APPROVED** [review]

File: Containerfile (hunk 1 above)
Comment (mabulgu) [reply Containerfile:2]: I agree. however my aim is keeping the changes in the container file as min as it needed. once we have these pipelines running we can focus on the other things I guess by improving the Containerfile and stuff around those.

**Pull request merged**